contracts/test/TestContracts/TroveManagerTester.t.sol

```diff
@@ -103,6 +103,17 @@ contract TroveManagerTester is ITroveManagerTester, TroveManager {
         return LiquityMath._computeCR(_coll, _debt, _price);
     }
 
+    function getCollGasCompensation(uint256 _entireColl, uint256 _entireDebt, uint256 _boldInSPForOffsets)
+        external
+        pure
+        returns (uint256)
+    {
+        if (_boldInSPForOffsets < _entireDebt) {
+            _entireColl = _entireColl * _boldInSPForOffsets / _entireDebt;
```

Comment:
Nitpick: not technically `_entireColl` any more after this op, right? (but the result passed down is correct)